sdk/src/test/java/io/opentelemetry/sdk/OpenTelemetrySdkTest.java

```diff
@@ -19,21 +19,18 @@
 import static com.google.common.truth.Truth.assertThat;
 
 import io.opentelemetry.OpenTelemetry;
-import io.opentelemetry.sdk.distributedcontext.DistributedContextManagerSdk;
-import io.opentelemetry.sdk.metrics.MeterSdk;
-import io.opentelemetry.sdk.trace.TracerSdk;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
 @RunWith(JUnit4.class)
-public class OpenTelemetryTest {
```

Comment:
Is this file moved or copied?

Reply:
Renamed ;)

Reply:
Super confusing, looks like a move in the review tool.

Reply:
Yeah, agreed. I had the same problem when I re-reviewed my changes here ;)